.gitignore

```diff
@@ -1,3 +1,4 @@
 /composer.lock
 /vendor
 /parameters.yml
+/esb.yml
```

Comment:
There is no need to ignore the `esb.yml` file. That file is meant to be used in the root of project which depends on `webgriffe/esb`. So it shouldn't be inside the `webgriffe/esb`. Do you agree?

Reply:
Yes it's a little bit confusing. We plan to provide an ESB project skeleton to be able to do something like `composer create-project webgriffe/esb-project` and have all the required stuff already checked out. Anyway we're currently in a super-alpha state so we prefer to not rely on the current implementation to much.